middleware/app/Models/ProviderUserResource.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class ProviderUserResource extends Model
+{
+    use HasFactory;
+    protected $fillable = [
+        "user_id",              // ProviderUser ID
+        "provider_resource_id", // Course ID
+        "provider_id",          // Platform Provider ID
+        "completed",            // Enum (ProviderUserResourceStatus)
```

Comment:
This field should probably be updated to be called 'status' instead of 'completed'.

Reply:
And update other places it is used of course.